sdk/e2e/09_deploy_router_with_route_name_path_std_ensembler_test.py

```diff
@@ -0,0 +1,123 @@
+import os
```

Comment:
Is this test necessary? I'm just thinking we don't include any E2E tests for all other ensembler types too. And, by this logic, shouldn't we also add an experiments_mapping ensembler test?

Reply:
It's a good question. Probably doesn't make sense to have a test for the "route name path" specifically. But we don't have any other e2e test for the standard ensembler at the moment. So I think it's useful to add this. Likewise, we have a test case internally using experiment_mappings, with the proprietary experiment engine.

Reply:
> And, by this logic, shouldn't we also add an experiments_mapping ensembler test?
> But we don't have any other e2e test for the standard ensembler at the moment.
We do!! There's an e2e test that uses the standard ensembler with the proprietary experiment engine (it's found in the blandly named `01_create_router_test` file). We can perhaps rename that `01` test with a more explicit name to make it appear more explicitly as a test using the standard ensembler with experiment mappings if that helps 🤔

> Is this test necessary?
Well, it is an SDK mirror of the `09_deploy_router_with_route_name_path_std_ensembler_test` that we introduced for the API and so far we've decided to maintain both sets of e2e tests.

Reply:
>  We do!! There's an e2e test that uses the standard ensembler with the proprietary experiment engine

It's using the docker ensembler, no?

https://github.com/caraml-dev/turing/blob/main/api/e2e/test/testdata/create_router_nop_logger_proprietary_exp.json.tmpl#L77

Reply:
Oh wait, oops I got ensemblers confused with the proprietary experiment engine 😅

Reply:
But it wouldn't take too much effort to set up an e2e test with the proprietary experiment engine and a standard ensembler 😅

Reply:
I think the e2e tests are not meant to test every combination of values (that's expected to be done by the unit/integration tests). So, IMO renaming this test to `09_deploy_router_with_std_ensembler_test`, to have it represent std ensembler in general, may suffice.

Reply:
Okay I've just renamed the test :)